test/e2e/tests/003-surveyInfoEdit.js

```diff
@@ -0,0 +1,54 @@
+import { button, clearTextBox, click, expectExists, writeIntoTextBox, hoverTextBox, toRightOf } from '../utils/api'
+import { clickHomeBtnEditSurveyInfo, waitForLoader, clickSidebarBtnHome, expectHomeDashboard } from '../utils/ui'
+
+const getElementRightOfLabel = ({ label }) => toRightOf(label)
+
```

Comment:
You don’t need this method

Reply:
I will move on the following PR because I have to use too